internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1002,7 +1002,7 @@ public void testReadCompressedData(CompressionStrategy compressionStrategy) thro
    * For the local P2P testing, need to setup two different directories and ports for the two Da Vinci clients in order
    * to avoid conflicts.
    */
-  @Test(timeOut = 2 * TEST_TIMEOUT, enabled = false)
+  @Test(timeOut = 2 * TEST_TIMEOUT, enabled = true)
```

Comment:
This doesn't work as expected when testing on local so we need to keep it `false`

Reply:
Should we find some multi-cluster test class and move this to the test class? Let me try original test in some other cluster as I expect without cluster discovery, the raw transport client should fail on other cluster...

Reply:
That's a good point actually. We can do it